### DISCUSSION TITLE
Split the OrderItem by Channel

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1227] [Q&A] [answered]
Is there any way we can split the order by channel where each channel would get their OrderItems to fulfil?

**Comment by michaelbromley** [comment]
No, this is not something that is currently possible. It _might_ be possible to create some implementation of this via a plugin, but I'm not 100% sure. It would need some exploration and prototyping.

**Comment by ramkishore-yerubandi** [comment, marked as answer]
ok. Can i create a partial fulfilment of an order from Channel Admin Login via API, provided he has access to Orders ?

**Reply by michaelbromley** [reply]
Yes, partial fulfillment is possible, but the default UI does not show which orderline is associated with which channel, so you might need to implement your own UI for that part.

**Reply by ramkishore-yerubandi** [reply]
I am thinking to get the ProdcutID from Orderline Item and from that pick up the channel id. Will build a custom UI. Hope this approach is fine?

**Reply by michaelbromley** [reply]
Yes, sounds like that could work.

**Reply by ramkishore-yerubandi** [reply]
thanks!
